package.json

```diff
@@ -1,6 +1,9 @@
 {
   "private": true,
   "name": "o-topper",
+  "dependencies": {
+    "@financial-times/n-map-content-to-topper": "^1.0.0"
```

Comment:
Unfortunately Origami components need to be developed using Bower dependencies.

`n-map-content-to-topper` is not available on Bower because the GitHub repository is not public. Would you be able to make the repository public?

Reply:
I've made the repo public now 😄 Once the PR in `origami-component-converter` is merged, this will allow me to merge this one?

Reply:
Some more steps need to be done also:
- We need to add a `bower.json` file to n-map-content-to-topper, I've made a PR for that but the build is failing --> https://github.com/Financial-Times/n-map-content-to-topper/pull/5

- Then we need to run `bower install n-map-content-to-topper --save` in this project and commit that change.

Reply:
Great, leave this to me!

Reply:
Hey @JakeChampion, so I read this in the Bower docs so is it correct that I've instead installed the GitHub repo, see https://bower.io/docs/creating-packages. 

Running `bower install n-map-content-to-topper --save` resulted in the following error: `ENOTFOUND Package n-map-content-to-topper not found`.

Reply:
Origami runs our own Bower Registry, which contains all public repositories on the Financial-Times and FTLabs GitHub organisations. It looks like it had an issue with n-map-content-to-topper because it was originally made internal. I'll look into this now to fix it :-)

Reply:
Thank you! Let me know when that's done and I'll install using `bower install n-map-content-to-topper --save`

Reply:
I've just got it to work and have added it to the bower.json file 👍